src/e2e/java/teammates/e2e/util/BackDoor.java

```diff
@@ -520,10 +520,10 @@ public static FeedbackSessionAttributes getSoftDeletedSession(String feedbackSes
 
         FeedbackSessionsData sessionsData = JsonUtils.fromJson(response.responseBody, FeedbackSessionsData.class);
         FeedbackSessionData feedbackSession = sessionsData.getFeedbackSessions()
-                        .stream()
-                        .filter(fs -> fs.getFeedbackSessionName().equals(feedbackSessionName))
-                        .findFirst()
-                        .orElse(null);
+                .stream()
+                .filter(fs -> fs.getFeedbackSessionName().equals(feedbackSessionName))
+                .findFirst()
+                .orElse(null);
```

Comment:
Unrelated change?

Reply:
I think it is because I realized it is the only method with a different indentation level. Should I not include it?

Reply:
This should normally count as unrelated change, but I can close one eye.